Src/BootCamp.Chapter/Lesson3.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace BootCamp.Chapter
+{
+    class Lesson3
+    {
+
+        public static void Demo()
+        {
+            Console.WriteLine("Demo works");
```

Comment:
Demo itself should call the functions.

Reply:
I think that is the intention, you just need some time to complete ;)

Reply:
I am unsure how to call a function within a function.  The tests says it cannot find the functions if I put them within demo()